src/imp/span_imp.js

```diff
@@ -109,11 +103,11 @@ export default class SpanImp {
 
     // Getter only. The GUID is immutable once set internally.
     guid() {
```

Comment:
I have mixed feelings about the "lightstep extensions" but for stability and (scarce) dev time reasons, I decided to minimize the number of API changes, extensions or not.